packages/amplify-util-mock/src/storage/storage.ts

```diff
@@ -0,0 +1,126 @@
+import { AmplifyStorageSimulator } from 'amplify-storage-simulator';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import { getAmplifyMeta, addCleanupTask, getMockDataDirectory } from '../utils';
+import { ConfigOverrideManager } from '../utils/config-override';
+
+export class StorageTest {
+    private storageName: string;
+    private storageSimulator: AmplifyStorageSimulator;
+    private configOverrideManager: ConfigOverrideManager;
+    private storageRegion: string;
+    private bucketName: string;
+
+    async start(context) {
+        // loading s3 resource config form parameters.json
+        const meta = context.amplify.getProjectDetails().amplifyMeta;
+        const existingStorage = meta.storage;
+        this.storageRegion = meta.providers.awscloudformation.Region;
+        //console.log("existingStorage",existingStorage);
+        if (existingStorage === undefined || Object.keys(existingStorage).length === 0) {
+            return context.print.warning('Storage has not yet been added to this project.');
+        }
+        let backendPath = context.amplify.pathManager.getBackendDirPath();
+        const resourceName = Object.keys(existingStorage)[0];
+        const parametersFilePath = path.join(
+            backendPath,
+            'storage',
+            resourceName,
+            'parameters.json'
+        );
+
+        const localEnvFilePath = context.amplify.pathManager.getLocalEnvFilePath();
+        const localEnvInfo = context.amplify.readJsonFile(localEnvFilePath);
+        const storageParams = context.amplify.readJsonFile(parametersFilePath);
+        this.bucketName = `${storageParams.bucketName}-${localEnvInfo.envName}`
+        const route = path.join(
+            '/',
+            this.bucketName,
+            '/'
+        );
+
+        let localDirS3 = this.createLocalStorage(context, resourceName);
+        const port = 20005; // port for S3
```

Comment:
Move this to the top of the file so its easy to change

Reply:
done